src/mk.sh

```diff
@@ -525,6 +525,14 @@ function mk_build()
   IFS=' ' read -r -a options <<< "$raw_options"
   for option in "${options[@]}"; do
     case "$option" in
+      --info|-i)
+        local kernel_version=$(get_kernel_release "$flag")
+        local compiled_modules=$(cat modules.order | wc -l)
+        
+        echo -e "Release: $kernel_version"
+        echo -e "Total modules to be compiled: $compiled_modules"
```

Comment:
Why not use the function `say`?

Reply:
I didn't really think this through, I guess it does make sense to use `say`. Thanks!